lib/cuda/chordMVPSetup.cpp

```diff
@@ -0,0 +1,110 @@
+#include "chordMVPSetup.hpp"
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+
+REGISTER_CUDA_COMMAND(chordMVPSetup);
+
+chordMVPSetup::chordMVPSetup(Config& config, const std::string& unique_name,
```

Comment:
I'd like to keep this for now, so that the MVP continues to work -- and then the PR that removes this and replaces it with the new thing will make more sense and have greater glory!